source/install/self-managed-changelog.md

```diff
@@ -18,7 +18,7 @@ Lastest Mattermost Releases:
 **Release Day: 2021-07-16**
 
 ### Deprecations
- - The ``platform`` binary and “--platform” flag have been deprecated. If you are using the “--platform” flag or are using the ``platform`` binary directly to run the Mattermost server application via a systemd file or custom script, you will be required to use only the mattermost binary.
+ - The ``platform`` binary and “--platform” flag have been deprecated.
```

Comment:
I was actually thinking of replacing the word "deprecated" in favor of "removed". The rest of the message is fine. 
I guess the point I'd like to pass is that this is not a simple deprecation warning. That went out at the begin of the year if I remember correctly. Now we acted on the deprecation and completely removed the binary.